cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBridgeClusterTest.java

```diff
@@ -1096,18 +1097,39 @@ public void testProbeConfiguration()   {
     }
 
     @ParallelTest
-    public void testTracingConfiguration() {
-        KafkaBridge resource = new KafkaBridgeBuilder(this.resource)
-                .editSpec()
-                    .withJaegerTracing(new JaegerTracing())
-                .endSpec()
-                .build();
+    public void testJaegerTracingConfiguration() {
+        testTracingConfiguration(JaegerTracing.TYPE_JAEGER);
+    }
+
+    @ParallelTest
+    public void testOpenTelemetryTracingConfiguration() {
+        testTracingConfiguration(OpenTelemetryTracing.TYPE_OPENTELEMETRY);
+    }
+
+    private void testTracingConfiguration(String type) {
+
+        KafkaBridgeBuilder builder = new KafkaBridgeBuilder(this.resource);
+        switch (type) {
+            case JaegerTracing.TYPE_JAEGER:
+                builder.editSpec()
+                            .withJaegerTracing(new JaegerTracing())
+                        .endSpec();
+                break;
+            case OpenTelemetryTracing.TYPE_OPENTELEMETRY:
+                builder.editSpec()
+                            .withOpenTelemetryTracing(new OpenTelemetryTracing())
+                        .endSpec();
+                break;
+            default:
+                throw new IllegalArgumentException("The '" + type + "' is not a valid tracing type");
+        }
+        KafkaBridge resource = builder.build();
 
         KafkaBridgeCluster kb = KafkaBridgeCluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, resource, VERSIONS);
         Deployment deployment = kb.generateDeployment(new HashMap<>(), true, null, null);
         Container container = deployment.getSpec().getTemplate().getSpec().getContainers().get(0);
 
-        assertThat(AbstractModel.containerEnvVars(container).get(KafkaBridgeCluster.ENV_VAR_STRIMZI_TRACING), is("jaeger"));
+        assertThat(AbstractModel.containerEnvVars(container).get(KafkaBridgeCluster.ENV_VAR_STRIMZI_TRACING), is(type));
```

Comment:
If I got your comment right, are you referring to the other env vars that the user has to set through the `template` (i.e. JAEGER_SERVICE_NAME or OTEL_SERVICE_NAME and so on)? In this case we need to add them in the test code to add them but does it make sense? Or which config values are you referring to?

Reply:
I assume you are also passing some options for the bridge configuration? Or is this the only option? Check for example the Connect test which also checks the COnnect config for the interceptors etc.

Reply:
The bridge is different because based on the `bridge.tracing` property (that operator sets through `STRIMZI_TRACING` env var), it's the bridge code itself setting the interceptor properties with corresponding classes. So nothing we can check at this stage of the test. I guess this is the reason why it wasn't there before even with just OpenTracing supported.

Reply:
Ok, in that case this is probably fine.